packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -20,6 +20,9 @@ import LiveMessage from '../Announcer/LiveMessage';
 /* @conditional-compile-remove(mention) */
 import { defaultOnMentionRender } from './MentionRenderer';
 import DOMPurify from 'dompurify';
+import { _AttachmentDownloadCardsStrings } from '../AttachmentDownloadCards';
```

Comment:
Not neccesary related to this PR but for @JoshuaLai  to confirm, do we still need the type of `_AttachmentDownloadCardsStrings` and the function of `useLocaleStringsTrampoline`?